db/migrate/20160218120630_add_subscribed_member_to_actions.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubscribedMemberToActions < ActiveRecord::Migration
+  def change
+    add_column :actions, :subscribed_member, :boolean, default: true
```

Comment:
Does subscribed_member being true mean that the action was made by a subscribed member, or that the action the member took subscribed them to the mailing list?